plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterConfiguration.qml

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 2.0
+import UM 1.3 as UM
+
+/**
+ * The MonitorPrinterConfiguration accepts 2 configuration objects as input and
+ * applies them to a MonitorBuildplateConfiguration instance and two instances
+ * of MonitorExtruderConfiguration. It's used in both the MonitorPrintJobCard
+ * component as well as the MonitorPrinterCard component.
+ */
+Item
+{
+    id: base
+
+    // Extracted buildplate configuration
+    property alias buildplate: buildplateConfig.buildplate
+
+    // Extracted extruder configuration for position 0
+    property var config0: null
```

Comment:
Why not use an array and subsequently use a repeater to draw the components?

Reply:
Smort.

Reply:
I didn't think too much about it since we only have dual extruder network enabled machines, but I'll add it in for future proofness.

Reply:
Fixed in 55554c6